primer/brand
# Prose component

## Summary

Implementation for the Prose component.

[Documentation](https://primer-699c1cd34f-26139705.drafts.github.io/components/Prose)
[Storybook](https://primer-699c1cd34f-26139705.drafts.github.io/storybook/?path=/story/components-prose--default)

## List of notable changes:

- **added** Prose component
- **added** tests for Prose component
- **added** documentation for Prose component

## What should reviewers focus on?

- Making sure that regardless of the content within the Prose component it looks visually representative of how we would want it in a production environment

## Steps to test:

1. Open the Prose component in CI-deployed preview environment
2. Go to Prose story in Storybook
3. Verify that Prose behaves as described in the following issue.

## Supporting resources (related issues, external links, etc):

- https://github.com/github/primer/issues/2357

## Contributor checklist:


## Reviewer checklist:


## Screenshots:
